src/test/java/javax/validation/ValidationTest.java

```diff
@@ -157,7 +167,7 @@ public CustomValidationProviderClassLoader(String... suffixes) {
 
 		@Override
 		public Enumeration<URL> getResources(String name) throws IOException {
-			CustomEnumeration<URL> customEnumeration = new CustomEnumeration<URL>();
+			CustomEnumeration<URL> customEnumeration = new CustomEnumeration<>();
```

Comment:
I suppose it's a change triggered by Eclipse on autosave (the configuration was changed when updating to Neon). While it's not a problem for a simple patch like that, it might be a bad idea to keep that for more complex patches.